devices/Bmxx80/ListBme680HeaterProfileConfig.cs

```diff
@@ -0,0 +1,513 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+// This is a template and not intended to be used as is
+// This template List<Something> will be concerted into ListSomething
+// Once generated, this will be usable as a normal ListSomething
+
+using Iot.Device.Bmxx80;
+using System;
+
+namespace System.Collections.Generic
+{
+    /// <summary>
+    /// List class for type Bme680HeaterProfileConfig that has been automatically generated
+    /// </summary>
+    public class ListBme680HeaterProfileConfig : IEnumerable
+    {
+        private ArrayList _list;
+
+        /// <summary>
+        /// Initializes a new instance of the System.Collections.Generic.List class that
+        /// is empty and has the default initial capacity.
+        /// </summary>
+        public ListBme680HeaterProfileConfig()
+        {
+            _list = new ArrayList();
+        }
+
+        /// <summary>
+        /// Initializes a new instance of the System.Collections.Generic.List class that
+        /// contains elements copied from the specified collection and has sufficient capacity
+        /// to accommodate the number of elements copied.
+        /// </summary>
+        /// <param name="collection">The collection whose elements are copied to the new list.</param>
+        /// <exception cref="System.ArgumentNullException">collection is null</exception>
+        public ListBme680HeaterProfileConfig(IEnumerable collection)
+        {
+            if (collection == null)
+            {
+                throw new ArgumentNullException($"collection is null");
```

Comment:
you can remove the text in all exception, gain of space on nano ;-)

Reply:
Changed all.